### DISCUSSION TITLE
余裕があれば実装したいこと、不具合メモ

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#200] [00: 未分類]
### 絶対直したい不具合等
- [x] アニメーションでブラウザ幅を飛び出る場合の処理（`overflow hidden`）
- [x] iPhoneスマホ、Chrome、Safariで確認したところ、ヒーローセクションの絵文字動いていない -> #208 
- [x] 背景絵文字のz-index、ホバーの問題

### 積極的に直したい
- [x] ヒーローセクションのボタンにURL追加 & ボタンをコンポーネントとして切り出す #203 

### 余裕があれば実装したいこと、最後の最後で良い直したい不具合
- [ ] ヒーローセクションの絵文字を大量にする
- [ ] 吹き出しコンポーネントを左右で分けて作る ref: #207

**Comment by kazzyfrog** [comment]
自分も今一通り確認しました！

あと、下記も追加した方がいいかもです！：
- ヒーローセクションの、２つのボタンにURLリンクを追加
- ボタンは、TOPページでも複数回使うので、コンポーネントとして切り出しても良いかも（ファイルは作成してあるので）

**Reply by pss-aileen** [reply]
ヒーローセクション...URL...すっかり抜けてました！
ボタンもそうですね、切り出した方が良いですね。ありがとうございます！

**Comment by kazzyfrog** [comment]
絵文字の表示に関しては、別のスレッドを立てて、自分も先日調べたこととかメモしました！
参考までに！ #202 

ちなみに、自分のiPhoneから確認したところ、safariではヒーローセクションの絵文字とアニメーションが適切に表示されていました！
PCだと、safariで表示されていない感じです！

**Reply by pss-aileen** [reply]
自分もiPhone（11 Pro）で確認しましたが、
ChromeもSafariも、
Noto Emoji適応される前の普通の絵文字が一瞬表示されたのですが
その後絵文字自体、何も表示されない...といった感じでした。

**Reply by pss-aileen** [reply]
PCでは、Chromeは大丈夫でしたが、safariはダメでした。

あと、ヒーローセクション以外の絵文字はPCもSPでも大丈夫でした！

**Reply by pss-aileen** [reply]
@kazzyfrog 

どれを優先的に作業していくか悩んでいるのですが、
吹き出しコンポーネント実装の後にこちらでも良いでしょうか？

**Reply by kazzyfrog** [reply]
> 吹き出しコンポーネント実装の後にこちらでも良いでしょうか？

もちろんです！１つずつやっていく感じで大丈夫ですよ👍

**Comment by kazzyfrog** [comment]
- ヒーローセクションのボタンにURL追加
- ボタンをコンポーネントとして切り出す

ボタンに関することは、下記で２つまとめてissue化しました！
→#203 

よろしければ、このイシュー自分がやりますよ！！

**Reply by pss-aileen** [reply]
issueありがとうございます！！
ではすみません、そちらの内容よろしくお願いいたします！
